ocs_ci/ocs/node.py

```diff
@@ -957,6 +965,96 @@ def delete_and_create_osd_node_vsphere_upi(osd_node_name, use_existing_node=Fals
     return new_node_name
 
 
+def delete_and_create_osd_node_ibm_power_upi(osd_node_name, use_existing_node=False):
```

Comment:
This function is the same as `delete_and_create_osd_node_vsphere_upi_lso`. So you can use it here or can export some of the code to make it more generic. If there are some special changes for IBM and the code can be different(now or in the future), we don't have to export the code and leave it as it is now.

Reply:
@yitzhak12 I borrowed code from delete_and_create_osd_node_vsphere_upi_lso. This method does quite a few things specific to vSphere. 1. It has facilities to support the usage of exiting nodes. 2. It adds a disk when new node is created. 3. Validation of the number of pvs, it is straightforward.

In the case of IBM Power, we will always add a node. When a node is created, a disk is created automatically. The number of pv validation is through a configurable value. 

Due to these reasons, I would like to retain the code as is. There are quite a few differences.

Reply:
Okay, np. Whatever you feel comfortable with.